src/components/Modals/DeleteContactModal.jsx

```diff
@@ -12,70 +12,68 @@ import DialogContent from '@mui/material/DialogContent';
 import DialogContentText from '@mui/material/DialogContentText';
 // Utility Imports
 import { runNotification } from '@utils';
-// Context Imports
-import { UserListContext } from '@contexts';
 // Component Imports
 import { FormSection } from '../Form';
 
 /**
- * DeleteClientModal Component - Component that allows users to delete other user's
+ * DeleteContactModal Component - Component that allows users to delete other user's
  * Pod URLs from a user's list stored on their own Pod
  *
- * @memberof Clients
- * @name DeleteClientModal
+ * @memberof Contacts
+ * @name DeleteContactModal
  */
 
```

Comment:
Think we could remove the space between the JSDoc and `DeleteContactModal` so that the JSDoc could associate with the component below it. Would probably need to include `@returns` and `@params` for the component and React props, respectively.

Reply:
fixed